modules/SiteHeader/assets/scripts/has-sticky-header.js

```diff
@@ -0,0 +1,8 @@
+function registerHasStickyHeaderActions() {
+    window.sitchco.hooks.addAction(window.sitchco.constants.SCROLL, () => {
```

Comment:
Would it be handy if the register function passed in argument references to the framework, so we could do `export function registerHasStickyHeaderActions({ hooks, constants }) { ... }`? That would cut down some bloat needing to globally reference everything from the framework.